src/constexpr_utils.hh

```diff
@@ -13,7 +13,7 @@ namespace Kakoune
 template<typename T, size_t N>
 struct Array
 {
-    constexpr size_t size() const { return N; }
+    static constexpr size_t size() { return N; }
```

Comment:
Not sure this is very elegant, even if it makes sense in theory.

Reply:
Alright, I buy that.

Reply:
Well, I guess it doesn't really make sense in theory, either.  Removed.